.changeset/twelve-kangaroos-carry.md

```diff
@@ -0,0 +1,17 @@
+---
+'@talend/react-components': major
+---
+
+Allow to customize drawer's subtitle tag
+
+**Breaking change :**
+Props `subtitleTagLabel` and `subtitleTagTooltip` are replaced by a props `subtitleTag`.
+
+Props `subtitleTag` has following shape :
+```
+{
+    label: PropTypes.string,
+    tooltip: PropTypes.string,
+    component: PropTypes.element,
```

Comment:
```suggestion
    tagType: keyof typeof TagVariant,
```

We don't actually want *any* type of component here. We just want to be able to pick a type of tag. The API should reflect that.

Reply:
This could also be optional and defaulted to... 'default'

Reply:
Two problems with that :

1. We are in Javascript here, not typescript
2. TagVariant is not exported by the module (see ui/packages/design-system/src/components/Tag/index.ts)

:/

Reply:
I'm talking about the component that is in JS. So i'm not able to specify `keyof typeof TagVariant`. Same thing about typing `subtitleTagVariant` with value-limited string list

Reply:
Oh yeah, my bad. Drawer is also a DS component so I assumed you were working on the newer one. 

You're free to export a `export const TagTypesArray = ['default', 'success'...]` if that helps. You could then do `PropTypes.oneOf(tagsTypesArray)`

Reply:
OK thank you